gpu/gpu.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env python
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2021 IBM
+# Author: Kalpana Shetty <kalshett@in.ibm.com>
+
+import os
+import shutil
+from avocado import Test, skipUnless
+from avocado.utils import build, process, distro, git
+from avocado.utils.software_manager import SoftwareManager
+
+IS_POWER_NV = 'PowerNV' in open('/proc/cpuinfo', 'r').read()
+
+class GPUTests(Test):
+    """
+    GPU test cases make use of sample test code provided by the
+    CUDA toolkil and other generic test suites cover functional tests.
+    Tester to download and setup CUDA stack which will take care of
+    installing nvidia driver too. 
+    """
+    @skipUnless(IS_POWER_NV,
+                "GPU tests are supported only on PowerNV(POWER9)")
```

Comment:
Here the check done for Power_NV. If this test case is for Power9 and POWER_NV then please add Power9 check also.

Reply:
Done